### ISSUE TITLE
Map misclick > restart loop

### ISSUE THREAD

**Issue by slartifer** [HughJeffner/FFRK-LabMem#137]
Last night I experienced a restart loop that at first looked similar to #131, but the cause was a different misclick: the bot somehow managed to move the party on the map screen.  I returned to find this:

![image](https://user-images.githubusercontent.com/17887799/150678682-696ba459-a41f-4b53-9976-4a92d0b5af2f.png)

I confirmed that reopening 'Labyrinth Dungeons' still leaves Tyro on the teleport hub.

I guess there must have been a similar issue with another button in the same location as that teleport hub, and the bot clicking here due to time elapsing between analysis and click.  Log from the first 2 restarts below; presumably the first one is where this occured.

[012322 misclick.txt](https://github.com/HughJeffner/FFRK-LabMem/files/7920490/012322.misclick.txt)

**Comment by HughJeffner** [comment]
Is there the possibility you:

- used the teleport node
- started a new lab run (without doing a QE first)
- have the `Use teleport stone when Master Painting is reached` control enabled?

**Comment by slartifer** [comment]
No chance I did anything -- this was immediately preceded by multiple bot runs while I was asleep.

I did enable the "use teleport stone when master..." control yesterday, though runs prior to this had no continuity issue when teleporting out.

**Comment by HughJeffner** [comment]
Ok. Well its a known issue if you warp to a teleport hub, then enter an adjacent lab dungeon and use a teleport stone Tyro will stay on the teleport hub and the bot has no way of getting back in.  Doing a QE will place Tyro on the dungeon node preventing the problem.

I was thinking maybe you were doing other lab content, and came back there to set the bot to run before bed?  It would complete at least 1 run before the problem showed up.

**Comment by slartifer** [comment]
OK, on closer examination it looks like this did occur on the first lab run of the sequence, not multiple runs.  I had interrupted the bot after a run to dispose of HE; I don't remember switching cloisters at that point, but I guess the simplest explanation is that I did.  Ah well.  I feel like it will be easy to forget to use a QE in that circumstance, so maybe it's just safer to leave the master option on.

**Comment by HughJeffner** [comment]
If I'm not mistaken, it would take 2 taps to navigate to the teleport node, 1 tap on the node and another to confirm the move.  The log fragment only shows 1 tap I think?

Since you are unsure, if you want we can further investigate this since the log has the pixel locations on screen it can be checked with the screen regions that would have to be tapped.

**Comment by slartifer** [comment]
That's right, just 1 tap.  This seems like a good confirmation of what happened.
